src/resources/js/components/AccountEdit.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <v-container>
+
+  <v-alert
+    v-model="message.show"
+    :class="message.type"
+    dismissible
+    >
+      {{ message.content }}
+    </v-alert>
+
+  <h2>My Account</h2>
+
+  <v-form
+    ref="form"
+    v-model="valid"
+    lazy-validation
+  >
+    <v-text-field
+      v-model="updateUserInfo.fname"
+      :counter="10"
+      :rules="nameRules"
+      label="firstName"
+    ></v-text-field>
+
+    <v-text-field
+      v-model="updateUserInfo.lname"
+      :counter="10"
+      :rules="nameRules"
+      label="lastName"
+    ></v-text-field>
+
+    <v-text-field
+      v-model="updateUserInfo.email"
+      :rules="emailRules"
+      label="E-mail"
+    ></v-text-field>
```

Comment:
same, nameRules should be in french and the tag

Reply:
nameRules isn't show to user, tag is corrected here [a71e2](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/123/commits/a71e249f6c074952cf0061a60c288e4f9804268d)